shared-module/bitmaptools/__init__.c

```diff
@@ -918,3 +921,59 @@ void common_hal_bitmaptools_alphablend(displayio_bitmap_t *dest, displayio_bitma
         }
     }
 }
+
+STATIC void draw_circle(displayio_bitmap_t *destination,
+    int16_t x0, int16_t y0,
+    int16_t radius, uint32_t value) {
+
+    int16_t d, y;
+
+    mp_arg_validate_int_range(x0, SHRT_MIN, SHRT_MAX, MP_QSTR_x0);
+    mp_arg_validate_int_range(y0, SHRT_MIN, SHRT_MAX, MP_QSTR_y0);
+
+    BITMAP_DEBUG("x, y, radius    (%4d, %4d, %4d)\n", x0, y0, radius);
+
+    y = radius;
```

Comment:
Is the value of `radius` ever checked? Just even to ensure it is not negative or 0.

Reply:
please see https://github.com/adafruit/circuitpython/pull/7782/commits/6ad053e40375169e907dc5f4949775bef3c6b332